src/Sylius/Bundle/AdminBundle/Resources/config/routing.yml

```diff
@@ -88,3 +88,8 @@ sylius_admin_partial:
 sylius_admin_api_taxon:
     resource: "@SyliusAdminBundle/Resources/config/routing/api/taxon.yml"
     prefix: /api
+
+sylius_admin_switch_locale:
```

Comment:
switch -> change? We are using this term in shop context. (locale/currency change handler, etc.)